tools/reactive-controllers/src/MatchMedia.ts

```diff
@@ -15,35 +15,153 @@ export const DARK_MODE = '(prefers-color-scheme: dark)';
 export const IS_MOBILE =
     '(max-width: 700px) and (hover: none) and (pointer: coarse), (max-height: 700px) and (hover: none) and (pointer: coarse)';
 
+export const MOBILE_SCREEN_QUERY = '(hover: none) and (pointer: coarse)';
+export const SMALL_SCREEN_QUERY = '(max-width: 743px)';
+export const LARGE_SCREEN_QUERY = '(min-width: 744px)'; // iPad Mini 6 max dimension, want it to be a 'tablet' using Desktop UI
+export const ORIENTATION_QUERY = '(orientation:portrait)';
+
 export class MatchMediaController implements ReactiveController {
     key = Symbol('match-media-key');
 
     matches = false;
 
     protected host: ReactiveElement;
 
-    protected media: MediaQueryList;
+    protected isMobileScreenQuery!: MediaQueryList;
+    protected smallScreenQuery!: MediaQueryList;
+    protected largeScreenQuery!: MediaQueryList;
+    protected orientationQuery!: MediaQueryList;
+
+    protected mediaQueryList!: MediaQueryList;
```

Comment:
nit: could we have a more descriptive name for this one? The name `mediaQueryList` confuses me a bit when reading through the code. Maybe something like `customMediaQueryList` would be clearer that this is used when a custom media query is passed?